packages/zoe/src/zoeService/zoe.js

```diff
@@ -62,6 +62,17 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
     return installation;
   };
 
+  /**
+   * @type {GetAmountOfInvitationThen}
+   */
```

Comment:
Is there a reason to prefer three line declarations? Line 76 uses one line for a similar declaration.  I admit that my prejudice is to get as much code on the screen at once as I can, and it doesn't seem that that's universal.

```suggestion
  /** @type {GetAmountOfInvitationThen} */
```

Reply:
Oh, no reason. I will change it

Reply:
Done